apps/i18n/common/en_us.json

```diff
@@ -2869,6 +2869,13 @@
   "progressV2_feedback_shareMoreLinkText": "Share more feedback",
   "progressV2_feedback_thumbsUp": "I like it!",
   "progressV2_feedback_thumbsDown": "I don't like it",
+  "submitProjectGallery_alreadySubmitted": "Submitted to Featured Projects gallery",
+  "submitProjectGallery_details": "*Submitting a project does not guarantee that it will be featured. Once submitted, this project cannot be submitted again.",
+  "submitProjectGallery_exampleText": "Example text",
+  "submitProjectGallery_goBack": "Go back",
+  "submitProjectGallery_header": "Want to show the world what you made?",
```

Comment:
We are trying to move away from full sentences in dialog headers.  This should be a shorter title, with a body text underneath if we want to ask a whole sentence.

Reply:
@samantha-code - lmk what you think.

Reply:
Maybe "Show the world what you made?" or "Show us what you made"

Reply:
I think 'Show us what you made' makes sense. @breville do you have a preference?